lib/cli/src/generators/configure.ts

```diff
@@ -0,0 +1,19 @@
+import fse from 'fs-extra';
+
+function mainConfigurationGenerator(addons?: string[], custom?: any) {
+  const hasSrc = fse.existsSync('./src');
+
+  const config = {
+    stories: [!hasSrc && '../stories/**/*.stories.*', hasSrc && '../src/**/*.stories.*'].filter(
```

Comment:
@ndelangen How does this fit with your fix on globs ?

Reply:
those are valid globs AFAICS